Header header

# текущая глубина аппарата, [м], [0, inf)
float64 distance

float64 velocity